engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -269,27 +269,15 @@ class EngineJobExecutionActor(replyTo: ActorRef,
       self,
       jobDescriptor,
       initializationData,
-      fileContentsHasherActor,
+      /*
+      Use context.system instead of context as the factory. Otherwise when we die, so will the child actors.
+       */
+      fileContentsHasherSingletonActor(backendName, factory, context.system),
```

Comment:
Can we somehow sneakily use the named `cromwell-root` actor to be the parent? @geoffjentry?